src/datasources/balances-api/zerion-balances-api.service.ts

```diff
@@ -92,6 +92,7 @@ export class ZerionBalancesApi implements IBalancesApi {
     safeAddress: `0x${string}`;
     fiatCode: string;
   }): Promise<Balance[]> {
+    // TODO: check the fiatCode is supported.
```

Comment:
This is not strictly related, but I've noted we make the request to Zerion when the `fiatCode` is invalid, so we should check it beforehand. I will do it in a separate PR if we are in an agreement.